charts/umbrella/README.md

```diff
@@ -1,24 +1,34 @@
+- [Umbrella Chart](#umbrella-chart)
+  - [Precondition for Semantic Hub](#precondition-for-semantic-hub)
```

Comment:
Putting this information at the beginning of the readme puts IMO a bit too much wight on it and might be confusing, could you please put it an at the end of usage after seeding?

Reply:
I moved that section to the bottom.